specification/deviceupdate/resource-manager/Microsoft.DeviceUpdate/preview/2020-03-01-preview/examples/CheckNameAvailability_Available.json

```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "api-version": "2020-03-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "locationName": "westus2",
+    "request": {
+      "name": "contoso",
+      "type": "Microsoft.DeviceUpdate/accounts"
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "nameAvailable": true,
+        "reason": "NotSpecified"
```

Comment:
>NotSpecified [](start = 19, length = 12)

remove

Reply:
I tried to call ARM for my type, which then I'd handled by RPaaS. That was the response.

Is it incorrect? Should I work with RPaaS to remove `reason` on success?